### ISSUE TITLE
MD045: Do not fire if aria-hidden=true

### ISSUE THREAD

**Issue by martincostello** [DavidAnson/markdownlint#1576]
If an inline image is specified as HTML to control sizing, MD045 fires even if the [`aria-hidden="true"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-hidden) attribute is included on the `img` element to hide the image from assistive technologies.

For example for a project logo:

```md
<img src="logo.png" width="200" height="200" aria-hidden="true" />
```

```text
MD045/no-alt-text: Images should have alternate text (alt text)
```

> This can improve the experience for assistive technology users by hiding:
>
> Purely decorative content, such as icons or images

In this context, the image is purely decorative, so there's no need to describe it as it can just be hidden.
